src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -354,6 +357,7 @@ internal static LanguageVersion RequiredVersion(this MessageID feature)
                 case MessageID.IDS_FeatureGenericAttributes: // semantic check
                 case MessageID.IDS_FeatureNewLinesInInterpolations: // semantic check
                 case MessageID.IDS_FeatureListPattern: // semantic check
+                case MessageID.IDS_FeatureRequiredMembers: // semantic check
```

Comment:
Why it's considered a semantic check?

Reply:
We try to make everything that can be a semantic check a semantic check to avoid invalidating incremental reparsing.